revpi-sos

```diff
@@ -5,29 +5,38 @@
 # Copyright 2020-2023 KUNBUS GmbH
 #
 
+sos_tmp_dir=/tmp
```

Comment:
Don't hardcode `/tmp`, use `$TMPDIR` and set `/tmp` as a fallback if the var is not set.

Additionally, global variables should be written in uppercase.

Reply:
That was my first approach `TMPDIR=${TMPDIR:-/tmp}`

But I changed it to this because:
-  The current implementation already used hardcoded value /tmp
- $TMPDIR is not set by default (on Debian/Raspbian)

I know it is considered best practise, the question here is, if the current implementation should be refactored or to keep the changes at necessary minimum

Reply:
I wrote my variables in lower_snake_case as most of the variables in the current implementation are written that way.

I'd personally prefer uppercase but as long as there are no formal conventions, I chose to stay consistent within this file